tests/extract_var_info_test.fish

```diff
@@ -2,6 +2,5 @@
 
 set --local variable "| a | b | c | d |"
 set --local actual (__fzf_extract_var_info variable (set --show | psub) | string collect)
-set --local expected "set in local scope, unexported, with 1 elements
-[1] | a | b | c | d |"
+set --local expected "set in local scope, unexported, with 1 elements"\n"[1] | a | b | c | d |"
```

Comment:
Made this one line after getting advice from Faho in https://gitter.im/fish-shell/fish-shell?at=6001f34191e9b71baddb5ffc